open-craft/opencraft
# feature: Enable customization of footer url and image

This adds the ability to customize the footer url and image which appears on the bottom right side of LMS pages (not to be confused with the organization url and images, found on the left side).

h1. Related Tickets

* [BB-4135](https://tasks.opencraft.com/browse/BB-4135)

h1. Testing instructions

1. Create a new instance through the registration client (I've already created https://stage.manage.opencraft.com/instance/7970/edx-appserver/3343/);
2. Change the "Footer Logo Image URL" and "Footer Logo Link URL" values in the "Logos" Theme configuration section (the preview should update automatically);
3. Publish the settings and wait until they finish applying;
4. Verify that the logo in the footer and the link changed correctly.
